src/Button/_variables.scss

```diff
@@ -0,0 +1,76 @@
+
+// Buttons + Forms
+//
+// Shared variables that are reassigned to `$input-` and `$btn-` specific variables.
+
+$input-btn-padding-y:         .375rem !default;
```

Comment:
Do these `$input-*` variables need to be here, or can they live in the Form folder?

Reply:
They are declared in both places unfortunately. They are shared variables. I could move them out to core instead, but since themes come before both they're still editable.

As I describe this is seems super wonky... I think I'll either depart from bootstrap and decouple them or move it to `core`